xls/dslx/lsp/dslx_ls.cc

```diff
@@ -249,6 +250,19 @@ absl::Status RealMain() {
         return std::vector<verible::lsp::InlayHint>{};
       });
 
+  dispatcher.AddRequestHandler(
+      "textDocument/documentHighlight",
+      [&](const verible::lsp::DocumentHighlightParams& params) {
+        auto highlights_or = language_server_adapter.DocumentHighlight(
+            params.textDocument.uri, params.position);
+        if (highlights_or.ok()) {
+          return highlights_or.value();
+        }
+        LspLog() << "could not perform document highlight(s); status: "
```

Comment:
I suspect that log was good for debugging but might be pretty noisy now ? Not all places the cursor passes by is a symbol.

Reply:
Testing it, looks like it is actually not an issue in practice.

Reply:
At least in VSCode it looks like it only happened on click. If it shows up as noisy in an editor down the road happy to remove.